pints/_plot.py

```diff
@@ -0,0 +1,248 @@
+#
+# Quick diagnostic tools for inference results
+#
+# This file is part of PINTS.
+#  Copyright (c) 2017, University of Oxford.
+#  For licensing information, see the LICENSE file distributed with the PINTS
+#  software package.
+#
+import pints
+import numpy as np
+from scipy import stats
+#import matplotlib.pyplot as plt
+import pylab as plt
+
+def trace(chain):
+    """
+    Trace plot.
+
+    `chain`
+        MCMC routine returned chain
+
+    """
+    n_sample, n_param = chain.shape
+
+    fig = plt.figure(figsize=(12, 2*n_param))
+    for i in xrange(n_param):
+        # Add histogram subplot
+        plt.subplot(n_param, 2, 1 + 2 * i)
+        plt.xlabel('Parameter ' + str(i + 1))
+        plt.ylabel('Frequency')
+        plt.hist(chain[:,i], label='p' + str(i + 1), bins=40, color='green',
+        alpha=0.5)
+
+        # Add trace subplot
+        plt.subplot(n_param, 2, 2 + 2 * i)
+        plt.xlabel('Iteration')
+        plt.ylabel('Parameter ' + str(i + 1))
+        plt.plot(chain[:,i], color='green', alpha=0.5)
+
+    plt.tight_layout()
+    return fig
+
+def autocorrelation(chain):
```

Comment:
I know I said we shouldn't but... can we have an argument for the maximum lag here?